docs/contributing/contributor_guide.md

```diff
@@ -96,7 +96,21 @@ Sometimes the Table of Contents, rendered on the right-hand side of pages, can g
 
 Prefer to use commas `,` as separators, quoting entries with double quotes `"` where necessary. These are required by the default table reader plugin settings. Formatting and style are not automated at this time.
 
-Most simple markdown formatting used within CSV Files will be rendered as expected. We encourage cross-linking from within tables to other parts of the documentation, or linking to external resources, as appropriate. Most VSCode markdown extension features do not function within CSV tables, and these are not linted at this time.
+Most simple markdown formatting used within CSV Files will be rendered as expected, while other complex markdown features might not display as expected. To enhance readability:
+
+- Avoid complex formatting and limit heavy styling inside CSV tables, as markdown linters often bypass content in these tables.
+- To control line breaks within table cells:
+    - Use `<br>` without surrounding spaces for a new line.
+    - Use `<br>&nbsp;` to create a new, indented line within the cell. Including the extra two spaces enhances consistency with unordered lists elsewhere in markdown.
+
```

Comment:
These blank lines between unordered list items will be removed when rendered. Let's remove them also in the source document.

Reply:
Alternatively, provide higher-level list items to group the existing list items.

Reply:
Fixed, Thank you.